README.md

```diff
@@ -176,6 +176,12 @@ file_paths = GranuleHandler.download_from_granules(granules, session=session)
 
 # Saves to current directory, use `path` to save somewhere else
 GranuleHandler.download_from_granules(granules, session=session, path="../Desktop")
+
+# Retrieve first approved types
+# Here, priority is given to "hdf", then "hf" etc.
+# Only the first approved link is downloaded
```

Comment:
My understanding of the iteration code is that the first link that fits the bill gets selected, and there's no priority to the the items in the tuple (since we are iterating through the links and not the `ext`)
```
        for link in granule.links:
            if link.href.host in [
                URLs.RESOURCE.value,
                URLs.NSIDC_RESOURCE.value,
                URLs.MOD11A2_V061_RESOURCE.value,
                URLs.LAADS_RESOURCE.value,
            ] and link.href.path.endswith(ext):
                return link.href
```

Reply:
Good catch, confirming that I'm seeing this behavior with debug tool:
![image](https://github.com/fraymio/modis-tools/assets/97044512/a2d8f8a2-8b0e-46e1-928c-d635517562d7)

From these possible links, both of these implementations will yield the `.nc` as it appears first in iterating the links:

`GranuleHandler.download_from_granules(lsn_granules, session, ("html", "nc", "xml"))`
or
`GranuleHandler.download_from_granules(lsn_granules, session, ("nc", "html", "xml"))`

![image](https://github.com/fraymio/modis-tools/assets/97044512/bf29debe-c4ab-4b02-b0fb-84fadcd48e0a)

I will update the docs accordingly.